### DISCUSSION TITLE
Multiply projects

### DISCUSSION THREAD

**Discussion by jelleruben** [opengisch/QField#5776] [Q&A]
Hello,

Is it possible to use and save multiply maps/projects?

I know if I load a map, it will be saved on the intern memory of my Android phone. But if I have 2 or 3 projects in Qfield. Are these also stored in separate folders, or is everything in 1 folder?

I hope you understand my question. 

Greetings, 

Jelle

**Comment by beanzmo** [comment]
Hi Jelle,
Have you looked at the QField Documentation https://docs.qfield.org/get-started/storage/
- The files will not automatically be stored, but you will have to save them to a directory. 
- Each project will be imported into a separate folder under 'imported projects' 

Does that make sense? 

B
